cli/internal/server/server_test.go

```diff
@@ -0,0 +1,72 @@
+package server
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/hashicorp/go-hclog"
+	"google.golang.org/grpc"
+	"gotest.tools/v3/assert"
+	"gotest.tools/v3/fs"
+
+	turbofs "github.com/vercel/turborepo/cli/internal/fs"
+)
+
+type mockGrpc struct {
+	stopped chan struct{}
+}
+
+func (m *mockGrpc) GracefulStop() {
+	close(m.stopped)
+}
+
+func (m *mockGrpc) RegisterService(desc *grpc.ServiceDesc, impl interface{}) {}
+
+func TestDeleteRepoRoot(t *testing.T) {
+	logger := hclog.Default()
+	repoRootRaw := fs.NewDir(t, "server-test")
```

Comment:
I don't *think* we're using the `"server-test"` portion of the path anywhere? If it doesn't matter, I prefer `t.TempDir()` because the method signature is infinitely more-clear than this random `fs` API that accepts a `testing` object.

```suggestion
	repoRootRaw := t.TempDir()
```

Reply:
(I'm not going to mark all of the ones throughout the rest of this PR as they'd just be noise and you'll know whether or not you have opinions on it from just this file.)

Reply:
Yeah, that seems fine.